cmd/tvx/stores.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"log"
+	"sync"
+
+	"github.com/fatih/color"
+	dssync "github.com/ipfs/go-datastore/sync"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/lib/blockstore"
+
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
```

Comment:
Should use `chain/actors/adt` to handle network upgrades correctly

Reply:
Resolved.